src/hotspot/share/classfile/vmSymbols.hpp

```diff
@@ -94,11 +94,32 @@
+  template(jdk_internal_vm_vector_VectorPayloadMF256B, "jdk/internal/vm/vector/VectorSupport$VectorPayloadMF256B") \
+  template(jdk_internal_vm_vector_VectorPayloadMF512B, "jdk/internal/vm/vector/VectorSupport$VectorPayloadMF512B") \
+  template(jdk_internal_vm_vector_VectorPayloadMF64S,  "jdk/internal/vm/vector/VectorSupport$VectorPayloadMF64S")  \
+  template(jdk_internal_vm_vector_VectorPayloadMF128S, "jdk/internal/vm/vector/VectorSupport$VectorPayloadMF128S") \
+  template(jdk_internal_vm_vector_VectorPayloadMF256S, "jdk/internal/vm/vector/VectorSupport$VectorPayloadMF256S") \
+  template(jdk_internal_vm_vector_VectorPayloadMF512S, "jdk/internal/vm/vector/VectorSupport$VectorPayloadMF512S") \
+  template(jdk_internal_vm_vector_VectorPayloadMF64I,  "jdk/internal/vm/vector/VectorSupport$VectorPayloadMF64I")  \
+  template(jdk_internal_vm_vector_VectorPayloadMF128I, "jdk/internal/vm/vector/VectorSupport$VectorPayloadMF128I") \
+  template(jdk_internal_vm_vector_VectorPayloadMF256I, "jdk/internal/vm/vector/VectorSupport$VectorPayloadMF256I") \
+  template(jdk_internal_vm_vector_VectorPayloadMF512I, "jdk/internal/vm/vector/VectorSupport$VectorPayloadMF512I") \
+  template(jdk_internal_vm_vector_VectorPayloadMF64L,  "jdk/internal/vm/vector/VectorSupport$VectorPayloadMF64L")  \
+  template(jdk_internal_vm_vector_VectorPayloadMF128L, "jdk/internal/vm/vector/VectorSupport$VectorPayloadMF128L") \
+  template(jdk_internal_vm_vector_VectorPayloadMF256L, "jdk/internal/vm/vector/VectorSupport$VectorPayloadMF256L") \
+  template(jdk_internal_vm_vector_VectorPayloadMF512L, "jdk/internal/vm/vector/VectorSupport$VectorPayloadMF512L") \
+  template(jdk_internal_vm_vector_VectorPayloadMF64F,  "jdk/internal/vm/vector/VectorSupport$VectorPayloadMF64F")  \
+  template(jdk_internal_vm_vector_VectorPayloadMF128F, "jdk/internal/vm/vector/VectorSupport$VectorPayloadMF128F") \
+  template(jdk_internal_vm_vector_VectorPayloadMF256F, "jdk/internal/vm/vector/VectorSupport$VectorPayloadMF256F") \
+  template(jdk_internal_vm_vector_VectorPayloadMF512F, "jdk/internal/vm/vector/VectorSupport$VectorPayloadMF512F") \
+  template(jdk_internal_vm_vector_VectorPayloadMF64D,  "jdk/internal/vm/vector/VectorSupport$VectorPayloadMF64D")  \
+  template(jdk_internal_vm_vector_VectorPayloadMF128D, "jdk/internal/vm/vector/VectorSupport$VectorPayloadMF128D") \
+  template(jdk_internal_vm_vector_VectorPayloadMF256D, "jdk/internal/vm/vector/VectorSupport$VectorPayloadMF256D") \
+  template(jdk_internal_vm_vector_VectorPayloadMF512D, "jdk/internal/vm/vector/VectorSupport$VectorPayloadMF512D") \
   template(payload_name,                              "payload")                                  \
+  template(mfield_name,                               "mfield")                                   \
   template(ETYPE_name,                                "ETYPE")                                    \
   template(VLENGTH_name,                              "VLENGTH")                                  \
```

Comment:
style: suggest to align with line-120.

```suggestion
  template(jdk_internal_vm_vector_VectorPayloadMF512D, "jdk/internal/vm/vector/VectorSupport$VectorPayloadMF512D") \
  template(payload_name,                               "payload")                                  \
  template(mfield_name,                                "mfield")                                   \
  template(ETYPE_name,                                 "ETYPE")                                    \
  template(VLENGTH_name,                               "VLENGTH")                                  \
```